src/platforms/android/manual-init.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Manual Initialization
```

Comment:
just a nit, so feel free to ignore if you think this isn't relevant.

this page could go under `platforms/android/configuration/` as it's not a feature or anything, just a configuration you can do, most of the people don't, I believe.

Reply:
I agree, upon reflection and deeper reading of what we're covering here. Thanks.